src/Components/Timer.tsx

```diff
@@ -80,7 +66,7 @@ const Timer = () => {
       pause();
       gameAudio.pause();
```

Comment:
this should pause the gameAudio.
If you wanna keep the setMusicOn(false) on the other file, just remove this line

Reply:
also line 56*, you can remove that line too